### DISCUSSION TITLE
不会写🍷别写

### DISCUSSION THREAD

**Discussion by codecow8** [vbenjs/vue-vben-admin#3855] [Q&A] [answered]
能不能别搞一堆无用封装

**Comment by mynameiscuining** [comment]
后端表示很难用，这些封装只有作者自己看的懂吧，学习成本高

**Comment by cn-src** [comment]
我感觉是存在过度封装。 

vben3还把 UI 库切换成了 naive-ui，仅仅是因为它是新的还是因为尤雨溪推荐过？

虽然 vben3 在代码结构上细化了些，但是感觉会比 vben2 更复杂。

**Comment by zrnwsy** [comment, marked as answer]
连一个组件都看不懂，看完还不会用，不找自己原因，净瞎bb，现在真是什么牛鬼蛇神都有，这是开源项目，作者没收钱，没逼着你用，你出门找个简单点的模板不就完事了

**Reply by cn-src** [reply]
稻草人谬误。懒得跟你扯。

**Reply by ChunyuPCY** [reply]
只有深入使用过的才知道 vben 这个模板的优秀，节省了很多的开发时间

**Comment by tjlizz** [comment]
一开始我也这么想，可用着用着就觉得挺好的。

虽说封装比较多，但你多看几次也了解差不多了。

一般情况下也不需要关心他怎么封装，看文档会用就行了呗。

自己去做一套试试才知道有多麻烦
